src/test/java/io/redis/examples/QueryFtExample.java

```diff
@@ -0,0 +1,315 @@
+            + "past graduation.\","
+            + "    \"condition\": \"refurbished\""
+            + "  }"
+        };
+
+        for (int i = 0; i < bicycleJsons.length; i++) {
+            jedis.jsonSet("bicycle:" + i, new Path2("$"), bicycleJsons[i]);
+        }
+
+        // STEP_START ft1
+        SearchResult res1 = jedis.ftSearch("idx:bicycle", "@description: kids");
+        System.out.println(res1.getTotalResults()); // >>> 2
+
+        List<Document> docs1 = res1.getDocuments();
+
+        for (int i = 0; i < docs1.size(); i++) {
+            System.out.println(docs1.get(i).getId());
+        }
+        // >>> bicycle:2
+        // >>> bicycle:1
+        // STEP_END
+
+        // Tests for 'ft1' step.
+        // REMOVE_START
+        Assert.assertEquals(2, res1.getTotalResults());
+        Assert.assertEquals("bicycle:2", docs1.get(0).getId());
+        Assert.assertEquals("bicycle:1", docs1.get(1).getId());
+        // REMOVE_END
+
+
+        // STEP_START ft2
+        SearchResult res2 = jedis.ftSearch("idx:bicycle","@model: ka*");
+        System.out.println(res2.getTotalResults()); // >>> 1
+
+        List<Document> docs2 = res2.getDocuments();
+
+        for (int i = 0; i < docs2.size(); i++) {
+            System.out.println(docs2.get(i).getId());
+        }
+        // >>> bicycle:4
+        // STEP_END
+
+        // Tests for 'ft2' step.
+        // REMOVE_START
+        Assert.assertEquals(1, res2.getTotalResults());
+        Assert.assertEquals("bicycle:4", docs2.get(0).getId());
+        // REMOVE_END
+
+
+        // STEP_START ft3
+        SearchResult res3 = jedis.ftSearch("idx:bicycle", "@brand: *bikes");
+        System.out.println(res3.getTotalResults()); // >>> 2
+
+        List<Document> docs3 = res3.getDocuments();
+
+        for (int i = 0; i < docs3.size(); i++) {
+            System.out.println(docs3.get(i).getId());
+        }
+        // >>> bicycle:6
+        // >>> bicycle:4
+        // STEP_END
+
+        // Tests for 'ft3' step.
+        // REMOVE_START
+        Assert.assertEquals(2, res3.getTotalResults());
+        Assert.assertEquals("bicycle:6", docs3.get(0).getId());
+        Assert.assertEquals("bicycle:4", docs3.get(1).getId());
+        // REMOVE_END
+
+
+        // STEP_START ft4
+        SearchResult res4 = jedis.ftSearch("idx:bicycle", "%optamized%");
+        System.out.println(res4.getTotalResults()); // >>> 1
+
+        List<Document> docs4 = res4.getDocuments();
+
+        for (int i = 0; i < docs4.size(); i++) {
+            System.out.println(docs4.get(i).getId());
+        }
+        // >>> bicycle:3
+        // STEP_END
+
+        // Tests for 'ft4' step.
+        // REMOVE_START
+        Assert.assertEquals(1, res4.getTotalResults());
+        Assert.assertEquals("bicycle:3", docs4.get(0).getId());
+        // REMOVE_END
+
+
+        // STEP_START ft5
+        SearchResult res5 = jedis.ftSearch(
+            "idx:bicycle",
+            new Query("%%optamised%%")
+        );
```

Comment:
Different query style than other queries. We should keep all the same.

Reply:
@sazzad16 It wasn't intentional - I just missed this one, but I've fixed it now. I think you said on another PR that FTSearchParams is generally better? Unless there is a situation where Query works better then I think we should just consistently avoid using it.